humblegen-rt/src/deser_helpers.rs

```diff
@@ -4,6 +4,20 @@ use crate::service_protocol::ErrorResponse;
 use crate::service_protocol::RuntimeError;
 use crate::service_protocol::ToErrorResponse;
 
+pub fn deser_param<T, E>(name: &str, value: &str) -> Result<T, ErrorResponse>
```

Comment:
Nit: shortening deserialize to deser seems really weird to me?

Reply:
follows the naming convention in the module, although it was me who established that naming convention in the first place :D